src/main/java/models/content/AnotherContent.java

```diff
@@ -0,0 +1,26 @@
+package models.content;
+
+import models.games.abstracts.AbstractContent;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AnotherContent extends AbstractContent {
+    public AnotherContent() {
+        List<String> pair = new ArrayList<>();
+        pair.add("No content!");
```

Comment:
Do we need 2 the same items in the list?

Reply:
Yes, it is necessary to have two any items in the list. In game.jsp are defined two Attributes - "answer1" and "answer2"